include/megaapi.h

```diff
@@ -6029,6 +6052,11 @@ class MegaGlobalListener
          *     After calling it, this callback will be called again with the corresponding
          *     state if there is really a change.
          *
+         *     - MegaApi::STORAGE_STATE_PAYWALL = 4
+         *     The account has been full for a long time. Now most of actions are disallowed.
+         *     It's needed to call MegaApi::getUserData to check the deadline/warnings
```

Comment:
```suggestion
         *     You will need to call MegaApi::getUserData before retrieving the deadline/warnings
```

Reply:
Plus , we might want to have some words in `getUserData` noting that will also update values such as those

Reply:
Ok